packages/assets/src/Assets.ts

```diff
@@ -611,9 +612,16 @@ export class AssetsClass
             loadWebFont,
         );
 
+        this.cache.addParser(
+            cacheSpritesheet,
+            cacheTextureArray
+        );
```

Comment:
All these parsers are only added on reset. Shouldn't they be added in the constructor as well?